services/catarse/app/models/subscription_monthly_report_for_project_owner.rb

```diff
@@ -43,8 +43,8 @@ def self.to_csv
 					sub.cpf,
 					sub.email,
 					sub.amount,
-					sub.service_fee,
-					sub.payment_method_fee,
+					sub.slip_fee > 0 ? (sub.payment_method_fee + sub.service_fee) : sub.service_fee,
+     				sub.slip_fee > 0 ? 0 : sub.payment_method_fee,
```

Comment:
**Problema:**
Não vi esse campo `slip_fee` sendo adicionado na subscription_monthly_report_for_project_owner, então isso vai dar erro na hora de gerar relatório, mas nem precisa adicionar, a taxa do boleto é só para projetos pontuais (que tem as contributions), tudo que for relacionado a subscription não precisa mudar, é o caso desse relatório aqui.